client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -1,98 +1,112 @@
-      (event) => {
-        const form = event.currentTarget;
-        if (form.checkValidity() === false) {
+    (event: React.FormEvent<HTMLInputElement>) => {
+      const form = event.currentTarget;
+      if (form.checkValidity() === false) {
         event.stopPropagation();
-        }
-        else {
-            onSet(calibValue);
-            setCalibDiff(calibValue-distTravelled);
-        }
-        event.preventDefault();
-        setValidation(true);
+      } else {
+        onSet(calibValue);
+      }
+      event.preventDefault();
+      setValidated(true);
     },
-    [calibValue, onSet, setValidation]);
+    [calibValue, onSet, setValidated],
+  );
 
-    const handleKeyPressed = useCallback(
-        (event) => { 
-            if (event.key === 'Enter') handleSubmit(event); 
-        },
-        [handleSubmit]
-    );
-
-    return (
-        <Card >
-            <Card.Body>
-                <Card.Title>Calibration</Card.Title>
-                <Card.Text>
-                    BOOST may use a distance different to the bike&apos;s travelled distance for the purposes of generating power plan data.
-                </Card.Text>
-                <div className="pb-3">
-                    <b>Distance travelled </b> <span className="float-right pr-4" > {distTravelled} m </span>
-                </div>
-                <div className="pb-3">
-                  <b>Calibrated distance </b> <span className="float-right pr-4"> {distTravelled + calibDiff} m </span>
-                </div>
-                <Form 
-                    noValidate 
-                    validated={validated} 
-                    onSubmit={handleSubmit} 
-                    onBlur={() => {setValidation(false);}}>
+  const handleKeyPressed = useCallback(
+    (event: React.KeyboardEvent<HTMLInputElement>) => {
+      if (event.key === 'Enter') handleSubmit(event);
+    },
+    [handleSubmit],
+  );
 
-                    <Form.Row>
-                        <Form.Group as={Col} md="4" >
-                            <Form.Control 
-                            type="number" 
-                            placeholder="Calibrate distance..." 
-                            onKeyPress={handleKeyPressed}
-                            onChange={handleCalibrationChange}
-                            min = "0"
-                            required/>
+  return (
+    <Card>
+      <Card.Body>
+        <Card.Title>Calibration</Card.Title>
+        <Card.Text>
+          BOOST may use a distance different to the bike&apos;s travelled
+          distance for the purposes of generating power plan data.
+        </Card.Text>
+        <div className="pb-3">
+          <b>Distance travelled </b>{' '}
+          <span className="float-right pr-4"> {distTravelled} m </span>
+        </div>
+        <div className="pb-3">
+          <b>Calibrated distance </b>{' '}
+          <span className="float-right pr-4">
+            {' '}
+            {distTravelled + calibrationDiff} m{' '}
```

Comment:
Nit: formatting the code maintained a bunch of spaces you had in the original, but it's made them explicit and a little ugly in the code. My guess is it'll look identical without the spaces. (I'm talking about the `{' '}` instances)

Reply:
Yeah it's quite annoying, the only reason why I hadn't removed them earlier was because whenever I reformat it's always going to bring them back unfortunately.

Reply:
That shouldn't be the case, they shouldn't appear again when deleted... Let me know if this keeps happening, interested to know why that would be the case